utils/errors/notifier/notifier.go

```diff
@@ -109,21 +122,25 @@ func convToSentry(in errors.ErrorExt) *raven.Stacktrace {
 	return out
 }
 
-func parseRawData(ctx context.Context, rawData ...interface{}) map[string]interface{} {
-	m := make(map[string]interface{})
+func parseRawData(ctx context.Context, rawData ...interface{}) (extraData map[string]interface{}, tagData []map[string]string) {
+	extraData = make(map[string]interface{})
+
 	for pos := range rawData {
 		data := rawData[pos]
 		if _, ok := data.(context.Context); ok {
 			continue
 		}
-		m[reflect.TypeOf(data).String()+strconv.Itoa(pos)] = data
+		if tags, ok := data.(isTags); ok {
```

Comment:
The tag data should be removed from other info present (in `rawData`).

This will prevent the same info being sent twice - once as part of tag data, once as part of extra data.

Reply:
makes sense.